private/src/scripts/editor/blocks/post-list/components/display/LinkList.jsx

```diff
@@ -1,6 +1,6 @@
 const LinkItem = (props) => (
   <li>
-    <article className="linkList-item">
+    <article className="linkList-item" style={{ backgroundImage: `url(${props.featured_image})` }}>
```

Comment:
if there's no featured image set, this will still output `style="background-image: url()"` in the DOM.

Reply:
Added conditional to check featured_image exists before outputting the background image styles